interfaces/Portal/src/app/program/program-people-affected/program-people-affected.component.ts

```diff
@@ -84,15 +84,16 @@ export class ProgramPeopleAffectedComponent implements OnDestroy {
   @Input()
   public programId: number;
   @Input()
-  public thisPhase: ProgramPhase;
+  public displayImportRegistration: boolean = false;
+  @Input()
```

Comment:
```suggestion
  public displayImportRegistration = false;

  @Input()
```

Hmm.. I thought ESLint would complain about this... :?
Not a real issues, but its not necessary to define the type as a `string`|`number`|`boolean` when you already set the default value (if that has one of these types)...